helix-term/src/commands.rs

```diff
@@ -1266,13 +1325,9 @@ fn find_char(cx: &mut Context, direction: Direction, inclusive: bool, extend: bo
             KeyEvent {
                 code: KeyCode::Enter,
                 ..
-            } =>
-            // TODO: this isn't quite correct when CRLF is involved.
-            // This hack will work in most cases, since documents don't
-            // usually mix line endings.  But we should fix it eventually
-            // anyway.
-            {
-                doc!(cx.editor).line_ending.as_str().chars().next().unwrap()
+            } => {
+                find_char_ret(cx, count, direction, inclusive, extend);
```

Comment:
minor nit: let's call this `find_line_ending` instead

Reply:
Specifically I'm looking for "line ending" to be in the name since we use that language consistently throughout the codebase & docs. `find_char_ending` I think is ambiguous but `find_char_line_ending` / `find_line_ending_char` etc would be good

Reply:
Oops 😅